hw11/hw1.py

```diff
@@ -0,0 +1,36 @@
+"""
+Vasya implemented nonoptimal Enum classes.
+Remove duplications in variables declarations using metaclasses.
+from enum import Enum
+class ColorsEnum(Enum):
+    RED = "RED"
+    BLUE = "BLUE"
+    ORANGE = "ORANGE"
+    BLACK = "BLACK"
+class SizesEnum(Enum):
+    XL = "XL"
+    L = "L"
+    M = "M"
+    S = "S"
+    XS = "XS"
+Should become:
+class ColorsEnum(metaclass=SimplifiedEnum):
+    __keys = ("RED", "BLUE", "ORANGE", "BLACK")
+class SizesEnum(metaclass=SimplifiedEnum):
+    __keys = ("XL", "L", "M", "S", "XS")
+assert ColorsEnum.RED == "RED"
+assert SizesEnum.XL == "XL"
+"""
+
+
+class SimplifiedEnum(type):
```

Comment:
if we modify it like this, we would lose possibility to iterate over enum

Reply:
But separated _dict_storage helps ups to avoid smth like "injections" when smb send us a tuple with standart class attributes. So I just added an iterator protocol realization